module/applications/config/action-config.mjs

```diff
@@ -62,24 +62,29 @@ export default class CrucibleActionConfig extends api.HandlebarsApplicationMixin
     description: {
       id: "description",
       template: "systems/crucible/templates/sheets/action/description.hbs",
+      scrollable: [""]
```

Comment:
I disagree with making the description tab scrollable. I agree that for other tabs, the tab itself should be scrollable, but I think scrolling on the description tab should come solely through the editor.